package.json

```diff
@@ -53,6 +56,7 @@
     "css-loader": "0.28.4",
     "dotenv": "4.0.0",
     "eslint": "4.4.1",
+    "eslint-config-airbnb": "^16.1.0",
```

Comment:
I'm getting a few peer dependency errors with this that won't allow me to run the lint script. Updating to the correct versions seemed to fix it.
`warning " > eslint-config-airbnb@16.1.0" has incorrect peer dependency "eslint@^4.9.0".
warning " > eslint-config-airbnb@16.1.0" has incorrect peer dependency "eslint-plugin-jsx-a11y@^6.0.2".
warning " > eslint-config-airbnb@16.1.0" has incorrect peer dependency "eslint-plugin-react@^7.4.0".`